sdk/storage/azure-storage-blob-changefeed/src/main/java/com/azure/storage/blob/changefeed/implementation/util/DownloadUtils.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.changefeed.implementation.util;
+
+import com.azure.storage.blob.BlobContainerAsyncClient;
+import reactor.core.publisher.Mono;
+
+import java.nio.charset.StandardCharsets;
+
+public class DownloadUtils {
+
+    /**
+     * Reduces a Flux of ByteBuffer into a Mono of String
+     */
+    public static Mono<String> downloadToString(BlobContainerAsyncClient client, String blobPath) {
+        return client.getBlobAsyncClient(blobPath)
+            .download()
+            .reduce(new StringBuilder(), (sb, buffer) -> {
+                sb.append(new String(buffer.array(), StandardCharsets.UTF_8));
```

Comment:
Would watch out on the usage of `ByteBuffer.array` as this will throw if the `ByteBuffer` is a `DirectByteBuffer`. May want to add in using `FluxUtil.byteBufferToArray`.

```java
sb.append(new String(FluxUtil.byteBufferToArray(buffer), StandardCharsets.UTF_8));
```